alf/examples/ac_breakout.gin

```diff
@@ -0,0 +1,53 @@
+# You need to install following packages
+# pip3 install atari-py opencv-python
+
+# environment config
+import tf_agents.environments.suite_atari
+# From OpenAI gym wiki:
+# "v0 vs v4: v0 has repeat_action_probability of 0.25
+#  (meaning 25% of the time the previous action will be used instead of the new action),
+#   while v4 has 0 (always follow your issued action)
+# Because we already implements frame_skip in AtariPreprocessing, we should always
+# use 'NoFrameSkip' Atari environments from OpenAI gym
+create_environment.env_name='BreakoutNoFrameskip-v4'
+create_environment.env_load_fn=@suite_atari.load
+create_environment.num_parallel_environments=512
+
+# Do not use this frame_skip! Potential issue with _pool_and_resize()
```

Comment:
I don't understand this comment. If it should not be used, why it is here.

Reply:
updated